extensions/remodeler/plugin-create-inline-properties.ts

```diff
@@ -288,8 +288,7 @@ function createVirtualParameters(operation: CommandOperation) {
         name: parameter.details.default.name,
         nameOptions: [parameter.details.default.name],
         description: parameter.details.default.description,
-        // required: true, /* if it's present in the variant, it's required  */
-        required: parameter.required, /* NEW: parameters pick up requiredness from the original operation. */
```

Comment:
@fearthecowboy out of curiosity -- is this `parameter.required` value not impacted by the `required` property set on a parameter in the spec? For example, does [this `required`](https://github.com/Azure/azure-rest-api-specs/blob/multiapi/specification/keyvault/data-plane/Microsoft.KeyVault/stable/7.0/keyvault.json#L996) have any impact on the value set for `parameter.required`?

Reply:
Well, theoretically yeah, but ... in OAI3 (which we upconvert to) ... the request body isn't a 'parameter' any more, it's 'the body' 

in the previous step, the 'parameter' that we create for 'the body' doesn't have any 'required' value.